### DISCUSSION TITLE
option to auto-connect on startup (windows)

### DISCUSSION THREAD

**Discussion by LitenBuzzTh** [hiddify/hiddify-app#637] [Q&A]
Hi
is there any specific settings to enable the hiddify windows app to connect the vpn as soon as it starts, and not to tap connect button on every windows start-up? (like nekoray and v2rayNG)

**Comment by volard** [comment]
As I got it, it's reviewed as a [feature request](https://github.com/hiddify/hiddify-app/issues/1426) and for now you [can do that with HiddifyCLI](https://github.com/hiddify/hiddify-app/issues/1504#issuecomment-2544759827)
